ui/src/app/cohort-review/detail-tab-table/detail-tab-table.component.tsx

```diff
@@ -0,0 +1,290 @@
+    box-shadow: 0;
+  }
+  body .p-paginator .p-paginator-pages .p-paginator-page.p-highlight {
+    background: #fafafa;
+    color: rgba(0, 0, 0, .5);
+  }
+  `;
+
+const styles = reactStyles({
+  table: {
+    fontSize: '12px',
+  },
+  tableBody: {
+    textAlign: 'left',
+    lineHeight: '0.6rem'
+  },
+  columnHeader: {
+    background: '#f4f4f4',
+    color: '#262262',
+    fontWeight: 600,
+  },
+  columnBody: {
+    background: '#ffffff',
+    padding: '5px',
+    verticalAlign: 'top',
+    textAlign: 'left',
+    borderLeft: 0,
+    borderRight: 0,
+    lineHeight: '0.6rem'
+  },
+  filterIcon: {
+    color: '#0086C1',
+    fontSize: '0.5rem',
+    float: 'right'
+  },
+  sortIcon: {
+    color: '#2691D0',
+    fontSize: '0.4rem'
+  }
+});
+
+export interface DetailTabTableProps {
+  tabname: string;
+  columns: Array<any>;
+  domain: string;
+  filterType: PageFilterType;
+  participantId: number;
+  workspace: WorkspaceData;
+}
+
+export interface DetailTabTableState {
+  data: Array<any>;
+  loading: boolean;
+  start: number;
+  rows: number;
+  sortField: string;
+  sortOrder: number;
+}
+
+export const DetailTabTable = withCurrentWorkspace()(
+  class extends React.Component<DetailTabTableProps, DetailTabTableState> {
+    dt: any;
+    constructor(props: DetailTabTableProps) {
+      super(props);
+      this.state = {
+        data: null,
+        loading: true,
+        start: 0,
+        rows: 25,
+        sortField: null,
+        sortOrder: 1
+      };
+    }
+
+    componentDidMount() {
+      this.getParticipantData();
+    }
+
+    componentDidUpdate(prevProps: any) {
+      if (prevProps.participantId !== this.props.participantId) {
+        this.setState({
+          data: null,
+          loading: true,
+        });
+        this.getParticipantData();
+      }
+    }
+
+    getParticipantData() {
+      const {cdrVersionId, id, namespace} = this.props.workspace;
+      const {cid} = urlParamsStore.getValue();
+      const pageFilterRequest = {
+        page: 0,
+        pageSize: 10000,
+        sortOrder: SortOrder.Asc,
+        sortColumn: this.props.columns[0].name,
+        pageFilterType: this.props.filterType,
+        domain: this.props.domain,
+      } as PageFilterRequest;
+
+      cohortReviewApi().getParticipantData(
+        namespace,
+        id,
+        +cid,
+        +cdrVersionId,
+        this.props.participantId,
+        pageFilterRequest
+      ).then(response => {
+        response.items.forEach(item => {
+          if (this.props.domain === DomainType[DomainType.VITAL]
+            || this.props.domain === DomainType[DomainType.LAB]) {
+            item['itemTime'] = moment(item.itemDate, 'YYYY-MM-DD HH:mm Z').format('hh:mm a z');
+          }
+          item.itemDate = moment(item.itemDate).format('YYYY-MM-DD');
+        });
+        this.setState({
+          data: response.items,
+          loading: false,
+        });
+      });
```

Comment:
We should have a `catch` to handle things in case the API call fails. (Even if it's just a placeholder for now)